CoreScriptsRoot/Modules/VR/VirtualKeyboard.lua

```diff
@@ -971,15 +968,7 @@ local function ConstructKeyboardUI(keyboardLayoutDefinitions)
 	end
 
 	local function UpdateTextEntryFieldText(newText)
-		local textSet = false
-		if useSetTextFromInput then
-			textSet = pcall(function() textEntryField:SetTextFromInput(newText) end)
-		end
-
-		if not textSet then
```

Comment:
Yeah, textset was just for seeing if the flag was set and SetTextFromInput succeeded. If not, do the old thing.